src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+}
+```
+
+Now, when you run `terragrunt plan`, Terragrunt will provide the input block's values, no longer prompting the command line for values.  Run `terragrunt plan`:
+
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] An execution plan has been generated and is shown below.
+[output] Resource actions are indicated with the following symbols:
+[output]  + create
+[output]
+[output] Terraform will perform the following actions:
+[output]
+[output]  # newrelic_alert_policy.DemoPolicy will be created
+[output]  + resource "newrelic_alert_policy" "DemoPolicy" {
+[output]    + account_id     = (known after apply)
+[output]    + id         = (known after apply)
+[output]    + incident_preference = "PER_POLICY"
+[output]    + name        = "My Demo Policy"
+[output]   }
+[output]
+[output] Plan: 1 to add, 0 to change, 0 to destroy.
+[output]
+[output] ------------------------------------------------------------------------
+[output]
+[output] Note: You didn't specify an "-out" parameter to save this plan, so Terraform
+[output] can't guarantee that exactly these actions will be performed if
+[output] "terraform apply" is subsequently run.
+```
+
+Run `terragrunt apply`. After Terraform finishes process, check your New Relic account for the new Demo Policy in your New Relic account.
+</Step>
+
+<!-- ADDING TO YOUR CONFIGURATION -->
+
+<Step>
+
+## Adding to your configuration
+
+Now that you have a basic New Relic configuration created, you add configurations built in the [Getting Started with Terraform](/terraform/get-started-terraform) and [Terraform modules](/terraform/terraform-modules) guides in this step.
+
+<Callout variant="tip">
+If you haven't done the previous developer guides, you can copy the configurations for this section from the <a href="https://github.com/jsbnr/nr-terragrunt-intro-example">Terragraunt Intro Github Repo</a>.
+</Callout>
+
+Update the __main.tf__ to include the addtional terraform configurations. Update the emaiil address in the alert channel to your prefered email address:
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	 name = "My Demo Policy"
+	}
+
+resource "newrelic_alert_channel" "DemoChannel" {
+	 name = "My Demo Channel"
+	 type = "email"
+
+	 config {
+	 recipients = "your@email_address.com"
+	 include_json_attachment = "1"
+	 }
+	}
+
+	resource "newrelic_alert_policy_channel" "ChannelSubs" {
+	 policy_id = newrelic_alert_policy.DemoPolicy.id
+	 channel_ids = [
+	 newrelic_alert_channel.DemoChannel.id
+	 ]
+	}
+
+	module "HostConditions" {
+	 source = "git::https://github.com/jsbnr/demo-terraform.git"
+	 policyId = newrelic_alert_policy.DemoPolicy.id
+	 cpu_critical = 88
+	 cpu_warning = 78
+	 diskPercent = 68
+	}
+```
+
+You have added a New Relic alert channel, subscribed the DemoPolicy to the alert channel, and added a module hosted on Github.
```

Comment:
[I noticed above](https://github.com/newrelic/developer-website/pull/1096/files#diff-cc8a9f5526b93409d5131c6ff2aaad33922d8d56764c68109dec2f80c02bb0a8R247) that `DemoPolicy` was 2 words. Should this also be 2 words? Or should the above reference change to a single word like this one? Or maybe these are indeed different?

Reply:
I would suggest:
```suggestion
You have added a New Relic alert channel, subscribed the demo policy to the alert channel, and added a module hosted on Github.
```